typescript/infra/config/environments/mainnet3/warp/eclipse-SOL-deployments.yaml

```diff
@@ -0,0 +1,23 @@
+# Configs and artifacts for the deployment of Hyperlane Warp Routes
```

Comment:
Did you craft these manually?

Reply:
Is there a way I can double check these configs? where did it come from?

Reply:
yeah I did, I crafted them using values from the registry. I've also ran with the configs so feel good they're accurate